keras/src/backend/jax/core.py

```diff
@@ -428,3 +428,13 @@ def device_scope(device_name):
     else:
         jax_device = device_name
     return jax.default_device(jax_device)
+
+
+def print(*args, **kwargs):
+    """Prints values and works in staged out JAX functions.
```

Comment:
The backend specific docstring is not visible anywhere (neither on keras.io, nor in an IDE). So move to the docstring of `keras/src/ops/core.py` explaining that this is JAX specific.